README.md

```diff
@@ -76,6 +76,28 @@ const { prettier } = require('@dreipol/pandora')
 module.exports = prettier.vue
 ```
 
+# React Setup
+
+## Installation
+
+For React projects we need to install the following two dependencies:
+
+```shell
+npm i -D eslint-plugin-react eslint-plugin-react-hooks
```

Comment:
Please add explicitly the packages versions compatible with pandora here. We want to be strict on that